iis-wf/iis-wf-referenceextraction/src/main/java/eu/dnetlib/iis/wf/referenceextraction/patent/PatentMetadataRetrieverJob.java

```diff
@@ -44,6 +53,8 @@
     private static final String COUNTER_PROCESSED_TOTAL = "processing.referenceExtraction.patent.retrieval.processed.total";
     
     private static final String COUNTER_PROCESSED_FAULT = "processing.referenceExtraction.patent.retrieval.processed.fault";
+    
+    private static final String COUNTER_FROMCACHE_TOTAL = "processing.referenceExtraction.patent.webcrawl.fromCache.total";
```

Comment:
Is `webcrawl` a good option here ? By comparing with `CachedWebCrawlerJob` I think `retrieval` would be better.

Reply:
No, definitely not. Blind copy&paste. Fixed here and in all the json files with reports.